scanner/cmd/scannerctl/main.go

```diff
@@ -55,7 +55,7 @@ func rootCmd(ctx context.Context) *cobra.Command {
 		"Address of the matcher service.")
 	serverName := flags.String(
 		"server-name",
-		"scanner-v4.stackrox",
```

Comment:
Will this break connect to a "combo mode" scanner (e.g., local scanner, testing) without specifying any `server name`?

If yes, can we change this so that [if only one connection is being made in the client](https://github.com/stackrox/stackrox/blob/master/pkg/scannerv4/client/client.go#L71), the default server name becomes "scanner-v4.stackrox"?

Reply:
I reverted that change and opted to add individual flags for indexer and matcher service names, similar to the address

Reply:
Those flags already exist, is that correct?

Example: https://github.com/stackrox/stackrox/blob/master/pkg/scannerv4/client/options.go#L83

Reply:
yes, I meant adding the flags to `scannerctl` (see below in this file)